.teamcity/src/main/kotlin/projects/CheckProject.kt

```diff
@@ -24,8 +25,21 @@ class CheckProject(
         // Avoid rebuilding same revision if it's already built on another branch (pre-tested commit)
         param("teamcity.vcsTrigger.runBuildOnSameRevisionInEveryBranch", "false")
         param("env.GRADLE_ENTERPRISE_ACCESS_KEY", "%ge.gradle.org.access.key%;%ge-td-dogfooding.grdev.net.access.key%")
-        param("additional.gradle.parameters", "")
-        param("reverse.dep.*.additional.gradle.parameters", "")
+
+        text(
+            "additional.gradle.parameters",
+            "",
+            display = ParameterDisplay.PROMPT,
```

Comment:
I don't think these should be prompt parameters, since that means that you always are asked to provide them.